xclim/indicators/atmos/_temperature.py

```diff
@@ -626,18 +626,83 @@ class TempWithIndexing(ResamplingIndicatorWithIndexing):
     standard_name="day_of_year",
     long_name="First day of year with temperature below {thresh}",
     description="First day of year with temperature below {thresh} for at least {window} days.",
-    compute=indices.first_day_below,
+    compute=indices.first_day_tn_below,
+    _version_deprecated="0.39.0",
 )
 
+first_day_tn_below = Temp(
+    identifier="first_day_below",
+    units="",
+    standard_name="day_of_year",
+    long_name="First day of year with minimum temperature below {thresh}",
+    description="First day of year with minimum temperature below {thresh} for at least {window} days.",
+    compute=indices.first_day_tn_below,
+    parameters=dict(thresh={"default": "0 degC"}, after_date={"default": "07-01"}),
+)
+
+
+first_day_tg_below = Temp(
+    identifier="first_day_below",
```

Comment:
All your indicators have the same identifier! It should be the same as the python name.

Reply:
Ah! Yes, absolutely. Good eye!